docs/sources/awscloudwatch.md

```diff
@@ -49,6 +49,13 @@ spec:
       name: default
 ```
 
+Alternatively you can use an IAM role for authentication instead of an access key and secret (Amazon EKS only):
+
+```yaml
+auth:
+  iamrole: arn:aws:iam::123456789012:role/foo
+```
+
```

Comment:
Not sure if we already have it somewhere, but do you think it worth putting the link to official doc about IAM roles support for SA ([link](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html))?
Asking because this one-liner hides a lot of configuration that have to be done by user on AWS side, leaving the impression that having any IAM role there is enough.

Reply:
Done